src/Microsoft.Health.Fhir.Api/Features/Routing/KnownRoutes.cs

```diff
@@ -37,6 +37,9 @@ internal class KnownRoutes
         public const string ValidateResourceType = ResourceType + "/" + Validate;
         public const string ValidateResourceTypeById = ResourceTypeById + "/" + Validate;
 
+        public const string Reindex = "$reindex";
+        public const string ReindexJobLocation = "$reindex" + "/" + IdRouteSegment;
```

Comment:
Should the operations be at the end? IdRouteSegment$reindex

Reply:
This should define the route as <server>/$reindex/idParameter, which should be correct.  Is that what you were asking?